### ISSUE TITLE
[NBug] Could not load file or assembly Microsoft.VisualStudio.Threading

### ISSUE THREAD

**Issue by gusarov** [gitextensions/GitExtensions.AzureDevOpsCommitMessage#1]
## Current behaviour

When I click "Plugin \ AzureDevOps" This error appears.
Also same happens every time I click "Commit" button, but the message there could be ignored and after that I can see commit dialog.

## Expected behaviour

no error message box 

## Steps to reproduce

Plugin \ AzureDevOps

## Error Details
```
System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.VisualStudio.Threading, Version=16.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
File name: 'Microsoft.VisualStudio.Threading, Version=16.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
   at GitExtensions.AzureDevOpsCommitMessage.Plugin.Execute(GitUIEventArgs args)
   at GitUI.CommandsDialogs.FormBrowse.<>c__DisplayClass63_0.<RegisterPlugins>b__2(Object <p0>, EventArgs <p1>)
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStripItem.FireEventInteractive(EventArgs e, ToolStripItemEventType met)
   at System.Windows.Forms.ToolStripItem.FireEvent(EventArgs e, ToolStripItemEventType met)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.ToolStripDropDown.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

WRN: Assembly binding logging is turned OFF.
To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There is some performance penalty associated with assembly bind failure logging.
To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

```

## Additional information
this always happens when trying to commit, also when open plugin -> AzureDevOps

## Environment
- Git Extensions 3.4.3.9999
- Build d4b0f48bbf3e39a71f5d7ff5231be5678d4aa0f1
- Git 2.27.0.windows.1
- Microsoft Windows NT 10.0.19041.0
- .NET Framework 4.8.4220.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
@pmiossec looks like the plugin is using an older version of Microsoft.VisualStudio.Threading, than is used in the app.

I _think_ this can be solved via a binding redirect to the current version in the plugin's config file.

**Comment by pmiossec** [comment]
>@pmiossec looks like the plugin is using an older version of Microsoft.VisualStudio.Threading, than is used in the app.

No, I think that @gusarov needs to update the plugin to version 1.1.0
I'm using it every days so it is working well...
And I also just checked and v1.1.0 is trying to load version 16.5.132.

>I think this can be solved via a binding redirect to the current version in the plugin's config file.

No. If I remember well, that doesn't work. The binding redirect need to be in the config file of the application. 

I already warned about this problem: https://github.com/gitextensions/gitextensions/issues/8003

That's why I was asking as a solution to add the binding redirect for all the assemblies of GitExtensions to reduce failing of assembly loading to only **really** breaking changes and not just version number changes!

Perhaps I need to answer to your last answer because I don't think it works like you described...
